frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -78,9 +123,37 @@ onUnmounted(() => {
     resetNotification();
 });
 
+function sortByNewUserAndUserName(
+    first: UserRoleOrAppAdminOrAccessControlPrivilegeWithIsNew,
+    second: UserRoleOrAppAdminOrAccessControlPrivilegeWithIsNew
+) {
+    // First, sort by isNewUser (true first)
+    if (first.isNewUser && !second.isNewUser) return -1;
+    if (!first.isNewUser && second.isNewUser) return 1;
+    return 0;
+}
```

Comment:
Sorry, maybe I didn't get it, where is the userName in the sorting? Could you add some function comment?

Reply:
I should have removed the userName from the name after I refactor it.